package.json

```diff
@@ -58,26 +58,42 @@
   "devDependencies": {
     "@types/jest": "^29.2.4",
     "@types/node": "^16.18.24",
-    "eslint": "^8.38.0",
-    "eslint-config-lostfictions": "^5.0.0",
-    "jest": "^29.5.0",
+    "@babel/preset-typescript": "^7.22.5",
+    "eslint": "^8.47.0",
+    "eslint-config-lostfictions": "^6.0.0",
+    "jest": "^29.6.2",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.1",
-    "ts-jest": "^29.1.0",
+    "ts-jest": "^29.1.1",
+    "babel-jest": "^29.6.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
+    "typescript": "^5.1.6",
     "zod": "~3.13.2"
   },
   "jest": {
     "preset": "ts-jest/presets/default-esm",
     "moduleNameMapper": {
-      "^(\\.{1,2}/.*)\\.js$": "$1"
+      "^(\\.{1,2}/.*)\\.c?js$": "$1"
     },
+    "moduleFileExtensions": [
+      "js",
+      "mjs",
+      "cjs",
+      "cts",
+      "jsx",
+      "ts",
+      "tsx",
+      "json",
+      "node"
+    ],
     "extensionsToTreatAsEsm": [
       ".ts"
     ],
     "rootDir": "src",
     "transform": {
+      "^.+\\.cts$": [
```

Comment:
The reason I had to add babel here is that `ts-jest` does not support the `cts` extension.
See https://github.com/kulshekhar/ts-jest/issues/3996